cpp/src/arrow/sparse_tensor_test.cc

```diff
@@ -26,6 +26,7 @@
 
 #include <gtest/gtest.h>
 
+#include <arrow/util/sort.h>
```

Comment:
Please leave includes in alphabetical order, and use quotes here: `#include "arrow/util/sort.h"`